src/prefect/environments/storage/docker.py

```diff
@@ -279,78 +355,14 @@ def create_dockerfile_object_from_base_image(
                 """.format(
                     base_image=self.base_image,
                     pip_installs=pip_installs,
+                    copy_flows=copy_flows,
                     copy_files=copy_files,
-                    flow_file_path=self.flow_file_path,
                     env_vars=env_vars,
                 )
             )
 
             dockerfile.write(file_contents)
 
-    def create_dockerfile_object_from_dockerfile(
```

Comment:
What's the intention behind removing the entire full dockerfile option? Too many extra required steps i.e. healthcheck, copying flows, etc...?

Reply:
So the intent was to simplify the interface for now, and the _only_ difference between `create_dockerfile_object_from_dockerfile` and the current method is that the current method's docker file is:
```
FROM {base_image}

# DO pip stuff
```
so I figured when we need this feature we can easily generalize our current method (and avoid having two methods).